lib/kubernetes-deploy/kubernetes_resource/service_account.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class ServiceAccount < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def sync
+      _, _err, st = kubectl.run("get", type, @name, "--output=json")
+      @found = st.success?
```

Comment:
Sorry I didn't notice last time, but we should set `@status` here too. I guess what Ingress has here would work fine for this case too: `@status = st.success? ? "Created" : "Unknown"`

That'll fix the summary:

![image](https://user-images.githubusercontent.com/4789493/28346645-50916a36-6c00-11e7-818f-9cc09261c064.png)